src/VisualStudio/Core/Next/FindReferences/StreamingFindReferencesPresenter.TableDataSourceFindReferencesContext.cs

```diff
@@ -499,9 +477,11 @@ private RoslynDefinitionBucket GetOrCreateDefinitionBucket(DefinitionItem defini
                 }
             }
 
-            public override void ReportProgress(int current, int maximum)
+            public override Task ReportProgressAsync(int current, int maximum)
             {
+                //var progress = maximum == 0 ? 0 : ((double)current / maximum);
```

Comment:
Yeah :(  The problem is that htsi method is still not threadsafe and will hang when we call it from the background.  Daofa will be fixing.

Note: a problem is that all users must then move to the build that has hte fix.  Otherwise, FAR will hang when we call this.  I'm not sure the best way to fix that. 

Currently, my approach is to just not implement this until enogh time has passed...